cmd/kubeadm/app/util/system/docker_validator.go

```diff
@@ -32,6 +31,14 @@ type DockerValidator struct {
 	Reporter Reporter
 }
 
+// dockerInfo holds a local subset of the Info struct from
+// github.com/docker/docker/api/types.
+// The JSON output from 'docker info' should map to this struct.
+type dockerInfo struct {
+	Driver        string
```

Comment:
shouldn't the struct contain the `json: "FieldName"` annotation

Reply:
Right. Should have the json tags. There will be deserializations.

Reply:
+1

Reply:
technically, it is not needed. the json deserializer in stdlib is happy if the struct field to match the name of the json field and it's not case sensitive. also we don't have serialization here.

but i will add the tags.